site/dat/docs/JMeter.md

```diff
@@ -205,6 +205,13 @@ scenarios:
         param2: value2
       body-file: path/to/file.txt  # this file contents will be used as post body
 
+      multipart-form: false  # use multipart/form-data for POST
```

Comment:
I think it is not needed. It's just to set "content-type" header. And usage of upload-files implies the multipart/form-data. I would set the content-type header automatically, but allow to override it if user would do so through "headers".